nncf/config/config.py

```diff
@@ -127,13 +128,17 @@ def _is_path_to_algorithm_name(path_parts: List[str]) -> bool:
     def validate(loaded_json):
         try:
             jsonschema.validate(loaded_json, NNCFConfig.schema())
+
+        except JsonSchemaValidationError as e:
+            logger.error('Invalid NNCF config supplied!')
```

Comment:
Why do you need this error handler? Isn't it the same as the next one? Shouldn't the next one executed?